src/main/java/com/book/dto/ProductsDTO.java

```diff
@@ -0,0 +1,14 @@
+package com.book.dto;
+
+import com.book.model.ProductsEntity;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ProductsDTO {
+    private List<ProductsEntity> productsEntityList;
```

Comment:
are u planning to have more paging APIs in the future? If yes, then u should create a `BaseDTO`object which has `currentPage`, `totalItems`, `totalPages`. 
Then all future DTO class will extend it